### PR TITLE
 Added support for 'isFinite', 'isInfinite', and 'isNaN'.

### PR DESCRIPTION
**Pull request by eaplatanios**
The names are not very swifty. @rxwei do you have any better suggestions? We could make them functions of `Tensor`.

### CODE REVIEW

File: Sources/TensorFlow/Operators/Math.swift [hunk 1]
```diff
@@ -686,6 +686,24 @@ func _vjpRelu<T: TensorFlowFloatingPoint>(
     return (relu(x), { v in Tensor(x .> 0) * v })
 }
 
+/// Checks which elements of `x` are finite.
+@inlinable
+public func isFinite<T: TensorFlowFloatingPoint>(_ x: Tensor<T>) -> Tensor<Bool> {
```
Comment (rxwei) [inline Sources/TensorFlow/Operators/Math.swift]: I think they should be defined as computed properties on `Tensor`. The names sound good as properties.

Comment (eaplatanios) [reply Sources/TensorFlow/Operators/Math.swift]: Sounds good. Done in the last commit.

**Comment by eaplatanios** [comment]
@rxwei I just updated this and all math tests pass locally, so it should be ready to merge. There are some failing tests related to lazy tensor that are not related to this PR.

**Comment by eaplatanios** [comment]
Also verified that all CI tests pass.

**Comment by eaplatanios** [comment]
@rxwei This is also merged as part of #157. I just did a minor cleanup of the tests here. Should I just merge this when I confirm CI tests pass?

**Comment by eaplatanios** [comment]
@rxwei Is it ok to directly merge minor fixes like this one as long as CI tests pass?

**Comment by rxwei** [comment]
Yes, thank you!

**Pull request merged**